homeassistant/components/binary_sensor/ffmpeg.py

```diff
@@ -73,8 +77,16 @@
 
 def setup_platform(hass, config, add_entities, discovery_info=None):
     """Create the binary sensor."""
-    from haffmpeg import SensorNoise, SensorMotion
+    from haffmpeg import Test, SensorNoise, SensorMotion
+
+    # check source
+    if config.get(CONF_RUN_TEST):
```

Comment:
It is not optional while it if default true. But you can disable it for faster start